app/components/PrimaryButton.tsx

```diff
@@ -15,7 +15,7 @@ export const PrimaryButtonStyle = StyleSheet.create({
   }
 })
 
-export function PrimaryButton (props: React.PropsWithChildren<ButtonProps> & { touchableStyle: StyleProp<ViewStyle> }): JSX.Element {
+export function PrimaryButton (props: React.PropsWithChildren<ButtonProps> & { touchableStyle?: StyleProp<ViewStyle> }): JSX.Element {
```

Comment:
need to disable injecting styles into components once we settle on the common DLS.

Reply:
Yep will do